helpers/entradaHelper/entrada.helper.go

```diff
@@ -287,24 +289,48 @@ func AddEntrada(data models.Movimiento) (result map[string]interface{}, outputEr
 				}
 
 				if err = request.SendJson(urlcrud, "POST", &resM, &movimientosKronos); err == nil {
-					// Cambia estado acta
+
+					// agrupa los elementos por clase para el asiento contable.
+					tipomvto := strconv.Itoa(data.IdTipoMovimiento)
+					var groups = make(map[int]float64)
+					for _, elemento := range actaRecibido.Elementos {
+						x := float64(0)
+						if val, ok := groups[elemento.SubgrupoCatalogoId]; ok {
+							x = val
+						}
+						groups[elemento.SubgrupoCatalogoId] = groups[elemento.SubgrupoCatalogoId] + x + elemento.ValorFinal
+					}
+					//fmt.Println(resA)
+					logs.Debug(resA)
```

Comment:
EN GENERAL
La idea es que los logs ayuden, evitar imprimir solamente las variables, hacerlos más descriptivos. Por ejemplo, no más indicar qué es lo que se está imprimiendo en consola, bastaría:
```suggestion
					logs.Debug("resA:", resA)
```